src/compressed_tensors/compressors/quantized_compressors/base.py

```diff
@@ -145,6 +147,17 @@ def compress(
 
         return compressed_dict
 
+    def _check_if_zp_pack_quantized(self, quant_args):
+        from compressed_tensors.compressors import PackedQuantizationCompressor
```

Comment:
This doesn't lead to a circular import? wonder if better off as a field on `PackedQuantizationCompressor`

Reply:
I kept it in the function to avoid potential circular imports